string label
float64 probability
uint16 centroid_x
uint16 centroid_y
uint16 left_bot_x
uint16 left_bot_y
uint16 right_top_x
uint16 right_top_y
